### ISSUE TITLE
Create wireframes for threads and comments

### ISSUE THREAD

**Issue by Skidle** [metisova/sturdy-lamp#28]
## Problem

It would be nice to have wireframes for the thread / comment related features before we start coding them.

## Solution suggestion

Continue with the same prototype, using material ui design with minimal alterations.

## Acceptance criteria

We have images added to the docs with a wireframe for each remaining feature:

- edit thread (if I'm an author)
- thread detail
- add comment
- edit comment
- comment tree in thread detail

## Learning outcome

Learned if a comment tree is actually achievable with material ui without custom components ✔️
